src/uu/cut/cut.md

```diff
@@ -3,7 +3,7 @@
 <!-- spell-checker:ignore sourcefile sourcefiles -->
 
 ```
-cut [-d|-w] [-s] [-z] [--output-delimiter] ((-f|-b|-c) {{sequence}}) {{sourcefile}}+
+cut OPTION... [FILE]...
```

Comment:
@tertsdiepraam for consistency, it should be [OPTION] here no?

Reply:
GNU does it without `[]`. I think that's because it requires at least one of the options.